commit 823a684f8d0495bd5e7b413e1a81fd5a600abef7
Author: Daniel P. Berrange <berrange@redhat.com>
Date:   Thu Feb 11 14:39:13 2010 +0000

    Fix USB device path formatting mixup
    
    * src/util/hostusb.c: The device path for a USB device wants the
      bus/device IDs in decimal not octal

diff --git a/src/util/hostusb.c b/src/util/hostusb.c
index 3cce66b..bf96539 100644
--- a/src/hostusb.c
+++ b/src/hostusb.c
@@ -184,7 +184,7 @@ usbGetDevice(unsigned bus,
     snprintf(dev->name, sizeof(dev->name), "%.3o:%.3o",
              dev->bus, dev->dev);
     snprintf(dev->path, sizeof(dev->path),
-             USB_DEVFS "%03o/%03o", dev->bus, dev->dev);
+             USB_DEVFS "%03d/%03d", dev->bus, dev->dev);
 
     /* XXX fixme. this should be product/vendor */
     snprintf(dev->id, sizeof(dev->id), "%d %d", dev->bus, dev->dev);
